src/Microsoft.IdentityModel.Tokens/Validation/Results/ValidatedToken.cs

```diff
@@ -14,30 +14,21 @@ namespace Microsoft.IdentityModel.Tokens
     /// <summary>
     /// Contains the results of successfully validating a <see cref="SecurityToken"/>.
     /// </summary>
-    internal class ValidatedToken
+    /// <remarks>
+    /// Creates an instance of <see cref="ValidatedToken"/>
+    /// </remarks>
+    /// <param name="securityToken">The <see cref="SecurityToken"/> that is being validated.</param>
+    /// <param name="tokenHandler">The <see cref="TokenHandler"/> that is being used to validate the token.</param>
+    /// <param name="validationParameters">The <see cref="ValidationParameters"/> to be used for validating the token.</param>
```

Comment:
We should comment that we throw if securityToken, tokenHandler, validationParameters are null.